EventListener/UserSessionListener.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file is part of the FOSUserBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\UserBundle\EventListener;
+
+use FOS\UserBundle\Event\GetResponseSessionUserEvent;
+use FOS\UserBundle\FOSUserEvents;
+use FOS\UserBundle\Model\UserInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+
+final class UserSessionListener implements EventSubscriberInterface
+{
+    /**
+     * @var UrlGeneratorInterface
+     */
+    private $router;
+
+    /**
+     * @var string
+     */
+    private $redirectRoute;
+
+    /**
+     * ResettingListener constructor.
+     *
+     * @param UrlGeneratorInterface $router
+     * @param string                $redirectRoute
+     */
+    public function __construct(UrlGeneratorInterface $router, $redirectRoute)
+    {
+        $this->router = $router;
+        $this->redirectRoute = $redirectRoute;
+    }
+
+    /**
+     * @return array
+     */
+    public static function getSubscribedEvents()
+    {
+        return array(
+            FOSUserEvents::RESETTING_RESET_REQUEST => 'onCheckSessionUser',
+            FOSUserEvents::RESETTING_RESET_INITIALIZE => 'onCheckSessionUser',
+            FOSUserEvents::RESETTING_SEND_EMAIL_INITIALIZE => 'onCheckSessionUser',
+            FOSUserEvents::REGISTRATION_INITIALIZE => 'onCheckSessionUser',
+            FOSUserEvents::REGISTRATION_CHECK => 'onCheckSessionUser',
+            FOSUserEvents::REGISTRATION_CONFIRM => 'onCheckSessionUser',
+            FOSUserEvents::SECURITY_LOGIN_INITIALIZE => 'onCheckSessionUser',
```

Comment:
this must be removed, as it breaks the usage of remember_me (which is one of the reason the previous PR was reverted).

and IMO, we should not have this behavior by default, and so we should not ship a listener (listener which does not support routes with a required parameter btw)